packages/tempus/src/Thyra_AdjointLinearOpWithSolveFactory.hpp

```diff
@@ -61,34 +61,26 @@ class AdjointLinearOpWithSolveFactory
     const RCP<const LinearOpWithSolveFactoryBase<Scalar> > &lowsf
     );
 
-  /** \brief . */
```

Comment:
I could never figure out why this was causing you problems.  You do have `EXTRACT_ALL` set to `YES` in the Tempus `Doxyfile`, though, so there is no need to include these comments to tell Doxygen to generate documentation for the methods&mdash;it should generate documentation for everything.